src/Admin/PostTable.php

```diff
@@ -63,9 +63,9 @@ public function table_column_content( $column, $post_id ) {
 				break;
 
 			case 'switch':
-				echo '<div class="onoffswitch" data-postid="' . esc_attr( $post_id ) . '" data-nonce="' . esc_attr( wp_create_nonce( 'change_notification_status_' . $post_id ) ) . '">';
-					echo '<input type="checkbox" name="notification_onoff_switch" class="onoffswitch-checkbox" value="1" id="onoffswitch-' . esc_attr( $post_id ) . '" ' . checked( $notification->is_enabled(), true, false ) . '>';
-					echo '<label class="onoffswitch-label" for="onoffswitch-' . esc_attr( $post_id ) . '">';
+				echo '<div class="onoffswitch" data-postid="' . esc_attr( (string) $post_id ) . '" data-nonce="' . esc_attr( wp_create_nonce( 'change_notification_status_' . $post_id ) ) . '">';
+					echo '<input type="checkbox" name="notification_onoff_switch" class="onoffswitch-checkbox" value="1" id="onoffswitch-' . esc_attr( (string) $post_id ) . '" ' . checked( $notification->is_enabled(), true, false ) . '>';
+					echo '<label class="onoffswitch-label" for="onoffswitch-' . esc_attr( (string) $post_id ) . '">';
```

Comment:
Why do you (convert&)escape integers?

Reply:
More of this case below.

Reply:
PHPStan and PHPCS made me do it ;)

If you don't escape PHPCS complaints. If you don't cast PHPStan complain

Reply:
Okay! Open a PHPCS issue.

Reply:
And please see the wonderful `tag()` function.
```php
echo tag(
'label',
[
    'class' => 'onoffswitch-label',
    'for' => sprintf('onoffswitch-%d', $post_id ),
],
$content
);
```